option optimize_for = LITE_RUNTIME;

message EquipSellRequest
{
 repeated uint32 pos = 1;
}
